vendors/infineon/boards/xmc4800_iotkit/aws_demos/config_files/FreeRTOSIPConfig.h

```diff
@@ -93,8 +93,8 @@ extern void vLoggingPrintf( const char * pcFormatString,
  */
 #define ipconfigUSE_DNS_CACHE                      ( 1 )
 #define ipconfigDNS_CACHE_ADDRESSES_PER_ENTRY      ( 6 )
-#define ipconfigDNS_CACHE_NAME_LENGTH              ( 16 )
-#define ipconfigDNS_CACHE_ENTRIES                  ( 4 )
+#define ipconfigDNS_CACHE_NAME_LENGTH              ( 81 )
```

Comment:
Why was 81 chosen for the name length?

Reply:
That is completely arbitrary. Add a comment like what for example?

Reply:
A number which is an exponential power of 2 (2,4,8,16,32 etc) or power of 2-1 (Something like 1,3,7,15,31 etc) seems okay when I see it as a constant somewhere.
But `81` seems something which is well thought out and must have an explanation for being there since it is so odd of a number. Therefore, I was looking for an explanation. 
But that just might be how I think :)

Reply:
Haha, 80 was the width in characters of the earliest text terminals. Plus a zero-byte makes 81. Later on, the lines became 128 bytes long.
The AWS IoT server that I use in my programs has a name of 46 characters. Making `ipconfigDNS_CACHE_NAME_LENGTH` equal to 64+1 would work for me as well, for now.

With this PR at least there will be a printed warning if the name doesn't fit.